.appveyor.yml

```diff
@@ -28,6 +28,7 @@ environment:
   PSQL_SSLMODE: disable
   PSQL_SKIPSQLCMD: true
   PSQL_TESTDBNAME: gct_dev_ci
+  CI_TEST: true
```

Comment:
I stole this: https://github.com/thrasher-corp/gocryptotrader/pull/1227#discussion_r1233480587